libs/booking/feature-seat-listing/src/lib/booking-feature-seat-listing.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'aap-passenger-info',
```

Comment:
```
  selector: 'app-passenger-info',
```

Reply:
off course, a pleasure to collaborate

Reply:
sorry i don't quite understand, you need me to change the prefix?
the app prefix is ​​'aap'. I think the prefix of this component could be 'aap-seat-listing' right?
if i use 'app' prefix 'tslint' will mark warning right?

Reply:
My bad, I didn't notice that @NachoVazquez set the prefix to `aap`. I wonder what that stands for 🤔